app/lib/server/startup/email.js

```diff
@@ -476,5 +476,9 @@ settings.addGroup('Email', function() {
 			type: 'boolean',
 			public: true,
 		});
+		this.add('Add_Sender_To_ReplyTo', false, {
+			type: 'boolean',
+			env: true,
```

Comment:
This `env` makes the setting value be set on `process.env` object, which I don't know how useful it is, maybe we should even remove that feature. for now I think we could prevent spreading it =)
```suggestion
```